project-fortis-interfaces/src/components/Admin/Admin.js

```diff
@@ -8,15 +8,17 @@ import TrustedSources from './TrustedSources';
 import { BlacklistEditor } from './BlacklistEditor';
 import StreamEditor from './StreamEditor';
 import AdminLocations from './AdminLocations';
+import UserRoles from './UserRoles';
 import '../../styles/Admin/Admin.css';
 
-const SETTINGS_TAB = 0;
-const WATCHLIST_TAB = 1;
-const LOCATIONS_TAB = 2;
-const CUSTOM_EVENTS_TAB = 3;
-const TRUSTED_SOURCES = 4;
-const BLACKLIST_TAB = 5;
-const STREAM_TAB = 6;
+const USERS_TAB = 0;
```

Comment:
I'd keep the settings tab as the first one -- it contains all the most important settings that the user has to change. Maybe put the users tab after the locations and before custom events.